### DISCUSSION TITLE
Welcome to TikTok-Live-Java Discussions!

### DISCUSSION THREAD

**Discussion by jwdeveloper** [jwdeveloper/TikTokLiveJava#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kohlerpop1** [comment]
Loving this project. It is coming in handy very much for my recent project, I am just looking for a few things.
1. Can more comments/docs be made above events without them to explain what they do and how they can be used? Some are very self explanatory while others have events, which may be useful, but are not explained or documented well.
2. Specifically, is there an event that indicates when boxes starts? I currently have lives/battles starting but I am looking for a way to detect the beginning of a boxes starting. 

Thanks for this again.
